.travis.yml

```diff
@@ -26,30 +30,30 @@ git:
 env:
   global:
     - ISSUES_REPO_URI=webcompat/webcompat-tests/issues
-    # grab latest 4.x release, which is LTS for now
-    - TRAVIS_NODE_VERSION="4"
 
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
 
 install:
-  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install node
```

Comment:
I think Travis has a nicer way to do this now, just using a `.nvmrc` file in the root.

https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions-using-.nvmrc

But maybe we just need the following, if we pick latest LTS and don't have to install nvm ourselves?

```
node_js:
    lts/*
```
maybe? Or maybe Travis will hate that, and we have to use .nvmrc. I don't know.

Reply:
Nice, I'll look into it.